cleverhans/attacks.py

```diff
@@ -749,6 +749,79 @@ def parse_params(self, y=None, y_target=None, nb_classes=None,
         self.clip_max = clip_max
 
 
+class DeepFool(Attack):
```

Comment:
There is no test for the new class. Could you add them to the `tests_tf/test_attacks.py` file?